src/common/libflux/rpc.c

```diff
@@ -37,6 +37,7 @@ struct flux_rpc {
     uint32_t matchtag;
     int flags;
     flux_future_t *f;
+    int sent:1;
```

Comment:
I was completely unaware of the `:` bit field syntax in a C struct.  Neat!

Any reason to use this over `bool`?  From the sparse info I can find online, it looks like bit fields have a higher runtime cost.

Reply:
Bitfields for boolean values may pack better in some cases, e.g. see  'struct job' [here](https://github.com/flux-framework/flux-core/blob/master/src/modules/job-manager/job.h) for one instance where I figured it might reduce memory footprint usefully.

In this case you cited, it's definitely not helping.  I'll switch to bool.